lib/ansible/plugins/httpapi/fortianalyzer.py

```diff
@@ -150,7 +150,14 @@ def send_request(self, method, params):
 
         try:
             if self.sid is None and params[0]["url"] != "sys/login/user":
-                raise FAZBaseException("An attempt was made to login with the SID None and URL != login url.")
+                # If not connected, send connection request.
+                if not self.connection._connected:
```

Comment:
`_connect` already does nothing if `_connected` is False. It might _look_ weird, but you can call `_connect()` without checking and the right thing will happen.

Reply:
10-4. Removed.

Reply:
Still waiting for it

Reply:
Should be fixed....